sermant-plugins/sermant-mq-grayscale/sermant-mq-grayscale-plugin/src/main/java/io/sermant/mq/grayscale/declarer/MqAbstractDeclarer.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.mq.grayscale.declarer;
+
+import io.sermant.core.plugin.agent.declarer.PluginDeclarer;
+import io.sermant.core.plugin.agent.declarer.SuperTypeDeclarer;
+
+/**
+ * abstract declarer
+ *
+ * @author chengyouling
+ * @since 2024-05-27
+ **/
+public abstract class MqAbstractDeclarer implements PluginDeclarer {
```

Comment:
AbstractPluginDeclarer  in core is same with MqAbstractDeclarer

Reply:
remove